.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!-- Please refer to our contributing documentation for any questions on submitting a pull request, or let us know here if you need any help: https://github.com/ionic-team/stencil/blob/master/.github/CONTRIBUTING.md -->
+
+## Pull request checklist
+
+Please check if your PR fulfills the following requirements:
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
+- [ ] Build (`npm run build`) was run locally and any changes were pushed
+- [ ] Unit tests (`npm test`) were run locally and passed
+- [ ] E2E Tests (`npm run test.karma.prod`) were run locally and passed
+- [ ] Prettier (`npm run format`) was run locally and passed
+
+## Pull request type
+
+<!-- Please do not submit updates to dependencies unless it fixes an issue. --> 
+
+<!-- Please try to limit your pull request to one type, submit multiple pull requests if needed. --> 
+
+Please check the type of change your PR introduces:
```

Comment:
A side effect of using a task list here is that all PRs will have "8 of 14 tasks" completed.